docker-compose.yml

```diff
@@ -14,8 +14,6 @@ services:
       POSTGRES_MULTIPLE_DATABASES: test,postgres
     ports:
       - "5432:5432"
-    volumes:
-      - postgres:/var/lib/postgresql/data
```

Comment:
Why was this removed?

Reply:
it caused some conflicts and it's already there few lines up ahead, during the last merge i mistakenly accepted both changes, so i had to remove one of them